src/adaptermanager.js

```diff
@@ -26,7 +26,14 @@ function getBids({ bidderCode, requestId, bidderRequestId, adUnits }) {
 }
 
 exports.callBids = ({ adUnits, cbTimeout }) => {
-  const requestId = utils.getUniqueIdentifierStr();
+  const requestId = utils.generateAuctionId();
+
+  const auctionInit = {
+    timestamp: Date.now(),
+    refererUrl: document.referrer,
```

Comment:
Removed. `referrer` is listed in the schema though, do we need to pass that from somewhere else?